pkg/dbutil/table.go

```diff
@@ -110,7 +114,10 @@ func GetTableInfoWithVersion(ctx context.Context, db QueryExecutor, schemaName s
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return GetTableInfoBySQL(createTableSQL, parser2)
+	sctx := mock.NewContext()
+	// unify the timezone to UTC +0:00
+	sctx.GetSessionVars().TimeZone = time.UTC
```

Comment:
we should use the same time zone of `createTableSQL`? which is the timezone of SQL connection

Reply:
we have already set session timezone to `+0:00`, so the `createTableSQL` gotten from session query is also base on `+0:00`.

https://github.com/pingcap/tidb-tools/blob/98f55c7bc75d1521bf634ae16ccfdf1ed9ae22b1/sync_diff_inspector/config/config.go#L186